server/ComputeSuperpixelFeatures/ComputeSuperpixelFeatures.xml

```diff
@@ -0,0 +1,190 @@
+      <longflag>reference_std_lab</longflag>
+      <default>0.57506023, 0.10403329, 0.01364062</default>
+    </double-vector>
+  </parameters>
+  <parameters advanced="true">
+    <label>Color Deconvolution</label>
+    <description>Color Deconvolution parameters</description>
+    <string-enumeration>
+      <name>stain_1</name>
+      <label>stain-1</label>
+      <description>Name of stain-1</description>
+      <channel>input</channel>
+      <longflag>stain_1</longflag>
+      <element>hematoxylin</element>
+      <element>eosin</element>
+      <element>dab</element>
+      <element>custom</element>
+      <default>hematoxylin</default>
+    </string-enumeration>
+    <double-vector>
+      <name>stain_1_vector</name>
+      <longflag>stain_1_vector</longflag>
+      <label>stain-1 custom vector</label>
+      <default>-1,-1,-1</default>
+      <description>Custom value for stain-1</description>
+    </double-vector>
+    <string-enumeration>
+      <name>stain_2</name>
+      <label>stain-2</label>
+      <description>Name of stain-2</description>
+      <channel>input</channel>
+      <longflag>stain_2</longflag>
+      <element>hematoxylin</element>
+      <element>eosin</element>
+      <element>dab</element>
+      <element>custom</element>
+      <default>eosin</default>
+    </string-enumeration>
+    <double-vector>
+      <name>stain_2_vector</name>
+      <longflag>stain_2_vector</longflag>
+      <label>stain-2 custom vector</label>
+      <default>-1,-1,-1</default>
+      <description>Custom value for stain-2</description>
+    </double-vector>
+    <string-enumeration>
+      <name>stain_3</name>
+      <label>stain-3</label>
+      <description>Name of stain-3</description>
+      <channel>input</channel>
+      <longflag>stain_3</longflag>
+      <element>hematoxylin</element>
+      <element>eosin</element>
+      <element>dab</element>
+      <element>null</element>
+      <element>custom</element>
+      <default>null</default>
+    </string-enumeration>
+    <double-vector>
+      <name>stain_3_vector</name>
+      <longflag>stain_3_vector</longflag>
+      <label>stain-3 custom vector</label>
+      <default>-1,-1,-1</default>
+      <description>Custom value for stain-3</description>
+    </double-vector>
+  </parameters>
+  <parameters advanced="true">
+    <label>WSI Analysis</label>
+    <description>Whole-slide image analysis (WSI) parameters</description>
+    <double>
+      <name>analysis_tile_size</name>
+      <label>Analysis Tile Size</label>
+      <description>Tile size for blockwise analysis</description>
+      <longflag>analysis_tile_size</longflag>
+      <default>4096</default>
+    </double>
+    <double>
+      <name>analysis_mag</name>
+      <label>Analysis Magnification</label>
+      <description>The magnification at which the analysis should be performed.</description>
+      <longflag>analysis_mag</longflag>
+      <default>20</default>
```

Comment:
Are you using 5x for your analysis? If so i would suggest setting that as default.